Troubleshooting-Doc/TroubleshootingDoc.adoc

```diff
@@ -0,0 +1,61 @@
+= Schulich Epsilon Software Troubleshooting Document
```

Comment:
Epsilon was a development name, let's go with the official car's name of Elysia for this document.